tokens.xml

```diff
@@ -11391,6 +11525,19 @@ Throne of the Dead Three — Reveal the top ten cards of your library. Put a cre
             <token>1</token>
             <tablerow>1</tablerow>
         </card>
+        <card>
+            <name>Saheeli, Filigree Master Emblem</name>
```

Comment:
```suggestion
            <name>Saheeli, Filigree Master (Emblem)</name>
```

Reply:
I thought we were using the name as it is in Scryfall (without parentheses) going forward

Reply:
then why do we still have parentheses in the file?

Reply:
Because I wasn't sure if I should take them all out or not. We had a short conversation in the dev channel about this where you argued for no parens, but no one else said anything so I wasn't sure if I should go in an take them all out or if we needed some kind of consensus first.